client/assets/css/style.css

```diff
@@ -0,0 +1,151 @@
+html {
+    font-family: CustomSansSerif,'Lucida Grande',Arial,sans-serif
+}
+
+body {
+    margin: 0;
+}
+
+.container {
+    width: 900px;
+    margin: auto;
+    padding: 21px 0;
+    position: relative;
+    display: flex;
+}
+
+.container .labels {
+    flex: 1
+}
+
+.container .labels div {
+    border-right: 1px solid rgba(0,0,0,.15);
+    text-align: right;
+    color: rgba(0,0,0,.44);
+    opacity: 0;
+}
+
+.container .labels .title-label {
```

Comment:
Some superfluous selectors here. Won't cause any issues but .title-label would be sufficient.